bitnami/external-dns/values.yaml

```
@@ -88,6 +88,9 @@ image:
   ##   - myRegistryKeySecretName
   ##
   pullSecrets: []
+## @param revisionHistoryLimit sets number of replicaset to keep in k8s
+##
+revisionHistoryLimit: 3
```

```suggestion
revisionHistoryLimit: 10
```

The default value is 10

https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#clean-up-policy

---

fixed